app/javascript/stylesheets/_base_layout.scss

```diff
@@ -113,6 +113,21 @@ a {
 	}
 }
 
+.dropdown-menu {
+  background-color: $primary;
+  &>li>a{
+      padding: 0.1em 0.5em;
+      display: block;
+      clear: both;
+      color: map-get($theme-text-colors, 'primary');
+      &:hover {
+        background-color:map-get($theme-text-colors, 'secondary');
+        color: map-get($theme-text-colors, 'primary');
```

Comment:
Flagging that the stylesheet includes an expectation that the primary and secondary text colors are high-contrast

Reply:
That is, that the two text colors are also a high-contrast pair

Reply:
good catch. i'm not sure i want to make this assumption. will want to find more appropriate existing/contrasty pair